Source/Classes/PINRemoteImageManager.m

```diff
@@ -784,13 +786,62 @@ - (void)downloadImageWithURL:(NSURL *)url
             NSError *remoteImageError = error;
             PINImage *image = nil;
             id alternativeRepresentation = nil;
-             
+            __block NSNumber *maxAge = nil;
```

Comment:
Any reason to box this as an NSNumber? If not, NSInteger should result shorter / faster code, as long as cases like NSNotFound or other sentinel values are handled.

Reply:
Yes, we need to distinguish between "set to 0" and "not set" (nil)